src/algorithms/time_evolution/evoltools.jl

```diff
@@ -181,27 +181,28 @@ Obtain the 3-site gate MPO on the southeast cluster at position `[row, col]`
         c      c+1
 ```
 """
-function _get_gatempo_se(gate::LocalOperator, row::Int, col::Int)
-    Nr, Nc = size(gate.lattice)
+function _get_gatempo_se(ham::LocalOperator, dt::Number, row::Int, col::Int)
+    Nr, Nc = size(ham.lattice)
     @assert 1 <= row <= Nr && 1 <= col <= Nc
-    unit = id(space(gate.terms[1].second, 1))
+    unit = id(space(ham.terms[1].second, 1))
```

Comment:
As a side note: this will fail if the spaces on different sites are different, which in principle would be allowed.
Not necessarily something that has to be fixed now, but good to keep in mind.

Additionally, `physicalspace(ham)[row, col]` is probably cleaner.